app/components/OfferRouteBox/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+*
+* OfferRouteComponent
+*
+*/
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import StyledSmall from 'components/shared/StyledSmall';
+import PageTitle from 'components/shared/PageTitle';
+
+import GuideCard from 'components/GuideCard';
+
+const TripBoxBorderDiv = styled.div`
+    box-shadow: 2px 4px 8px 0 rgba(0, 0, 0, 0.2);
+`;
```

Comment:
Instead of copy pasting this from wherever else it was, can you make a new component in the `components/shared` folder called "BorderOutlineDiv" or something? Consistency, reusability is crucial!